src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -21,54 +23,97 @@ type TableRowData = RowData & {
 
 // Columns
 
+function cellDefaultTooltip(displayValue: string) {
+  return <Tooltip title={displayValue}>{displayValue}</Tooltip>;
+}
+
+function cellNullTooltip() {
+  return (
+    <Tooltip title="NULL">
+      <i>NULL</i>
+    </Tooltip>
+  );
+}
```

Comment:
I don't think it makes a lot of sense to put `tooltip` in these functions' names.
The fact that they add a tooltip is not that meaningful. What is important is that they define a cell's content. They should be called just `nullCell | stringCell | rangeCell | ...`.